libs/libc/stdio/lib_libfread_unlocked.c

```diff
@@ -213,9 +215,9 @@ ssize_t lib_fread_unlocked(FAR void *ptr, size_t count, FAR FILE *stream)
                        * into the buffer.
                        */
 
-                      bytes_read = _NX_READ(stream->fs_fd,
-                                            stream->fs_bufread,
-                                            buffer_available);
+                      bytes_read = stream->fs_iofunc.read(stream->fs_cookie,
+                                              (char *)stream->fs_bufread,
```

Comment:
```suggestion
                                                             (char *)stream->fs_bufread,
```

Reply:
I can´t move it there, the line would be too long.

Reply:
ok

Reply:
but, need add FAR

Reply:
could we change fs_bufread type to char * to avoid the cast

Reply:
FAR added. I think I would rather cast it here than change the type through the entire NuttX now. Maybe in some further patch but not together with fopencookie.